code/queue.rb

```diff
@@ -455,6 +457,8 @@ def sublimate_config(conf)
       new_config.env_vars        = conf['env_vars']
       new_config.coalesce        = so_truthy?(conf['coalesce'])
       new_config.coalesce_params = Hash[ (1..4).map {|x| [x, so_truthy?(conf["coalesce_param#{x}"])]} ]
+      new_config.blocking        = so_truthy?(conf['blocking'])
+      new_config.blocking_params = Hash[ (1..4).map {|x| [x, so_truthy?(conf["blocking_param#{x}"])]} ]
```

Comment:
It's been on my TODO list to turn coalesce into an array, e.g. `coalesce_params: [1, 2]`. The old default for coalesce was `coalesce_param1: yes, coalesce: 0` which never made sense to me.

So for a new `blocking` option, I'd like to jump straight to a more sensible config, e.g. `blocking_params: [1, 2, 3, 4]`.